src/services/user.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable camelcase */
+const bcrypt = require('bcrypt');
+const database = require('../models');
+const registerValidation = require('../validation/user');
```

Comment:
@Osasbaby convert this service to class and let it have registerUser function that takes body as the argument